docs/11-custom-scripts.md

```diff
@@ -3,10 +3,41 @@ Custom scripts allow for additional cluster setup steps when the cluster is bein
 when you want to modify the default cluster configuration for things such as modifying spark.conf, adding jars
 or downloading any files you need in the cluster.
 
+You can specify the location of custom scripts on your local machine in `.thundebolt/cluster.yaml`. If you do not have a `.thunderbolt/` directory in you current working directory, run `azb spark init` or see [Getting Started](./00-getting-started). Note that the path can be absolute or relative to your current working directory.
+
+The custom scripts can be configured to run on the Spark master only, the Spark workers only, or all nodes in the cluster (Please note that by default, the Spark master node is also a Spark worker). For example, the following custom-script configuration will run 3 custom scripts:
+
```

Comment:
maybe worth pointing out here that the scripts will be executed in the order that the user defines it in the cluster.yaml

Reply:
I do that in the following paragraph, but it might be more clear to put that info here.

Reply:
yeah i noticed - what you have below is useful i think. But i think it could be stated more explicitly, like "scripts will execute in the order that they are listed".